configs/components/puppet-ca-bundle.rb

```diff
@@ -1,9 +1,25 @@
 component "puppet-ca-bundle" do |pkg, settings, platform|
   pkg.load_from_json("configs/components/puppet-ca-bundle.json")
+
   pkg.build_requires "openssl"
-  # facter will pull in java for server platforms, so require that so we can
-  # make the keystore
-  pkg.build_requires 'facter'
+
+  java_available = true
+  case platform.name
```

Comment:
I hate everything about the idea of remembering to maintain this list, but I don't have a better short-term idea right now